cmd/notation/plugin/install.go

```diff
@@ -138,10 +139,12 @@ func install(command *cobra.Command, opts *pluginInstallOpts) error {
 		}
 		defer os.Remove(tmpFile.Name())
 		defer tmpFile.Close()
+		fmt.Printf("Downloading plugin from %s\n", opts.pluginSource)
 		err = notationplugin.DownloadPluginFromURL(ctx, opts.pluginSource, tmpFile)
 		if err != nil {
 			return fmt.Errorf("failed to download plugin from URL %s with error: %w", opts.pluginSource, err)
 		}
+		fmt.Printf("Download completed. Saved to tmp file %s\n", tmpFile.Name())
```

Comment:
```suggestion
		fmt.Printf("Download completed. Saved to temporary file %s\n", tmpFile.Name())
```

Reply:
Do we really need to show the path of the temporary file as it will be deleted anyway?

Reply:
Removed.